packages/config/src/validation.ts

```diff
@@ -301,6 +301,16 @@ export function isNumberOrFalse (key: string, value: any): ErrResult | true {
   return errMsg(key, value, 'a number or false')
 }
 
+export function isValidCrfOrBoolean (key: string, value: any): ErrResult | true {
+  // a valid number that is between 1-51 including 1 or 51
+  // or a boolean. false disables compression and true sets compression to 32 CRF by default.
+  if (_.isBoolean(value) || (_.isNumber(value) && _.inRange(value, 0, 52))) {
+    return true
+  }
+
+  return errMsg(key, value, 'a valid CRF number between 0 & 51, false to disable compression, or true to use the default compression of 32')
```

Comment:
```suggestion
  return errMsg(key, value, 'a valid CRF number between 1 & 51, 0 or false to disable compression, or true to use the default compression of 32')
```

Reply:
This is where I am unsure. We don't compress the video if CRF is 0 but its the same thing as compressing a video with 0 CRF. Is this just going to be confusing?